### DISCUSSION TITLE
Display days between currentVersion and newVersion in Pull Request title or label

### DISCUSSION THREAD

**Discussion by hebestreit** [renovatebot/renovate#31228] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hi,

I would like to know if it's possible to display the difference in days between the current version and the new version which has been updated in the Pull Request? Showing this information inside the PR title or label helps us to identify packages that have not been updated for a long time.

Thanks in advance!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Not possible, also it would be very "noisy" with potentially daily PR title changes even if possible

**Comment by hebestreit** [comment]
@rarkins, thanks for your quick response and I agree. 

Would it be possible to make the `releaseTimestamp` property accessible in the template fields which we can be then added as a prefix to the PR title?

**Reply by rarkins** [reply]
Yes it could be added to https://github.com/renovatebot/renovate/blob/89b76373f738415486d2a57455b1104ec9a869db/lib/util/template/index.ts#L152, PR welcome

**Reply by hebestreit** [reply]
I've raised the PR and I'm looking forward to your feedback!

https://github.com/renovatebot/renovate/pull/31286
